static/docs/commands-reference/import-url.md

```diff
@@ -90,10 +90,10 @@ $ dvc run -d https://example.com/path/to/data.csv \
           wget https://example.com/path/to/data.csv -O data.csv
 ```
 
-Both methods generate an equivalent stage file (DVC-file) with an external
-dependency. The `dvc import-url` command saves the user from having to manually
-copy files from each of the remote storage schemes, and from having to install
-CLI tools for each service.
+Both methods generate an equivalent [stage file](/doc/commands-reference/run)
+(DVC-file) with an external dependency. The `dvc import-url` command saves the
+user from having to manually copy files from each of the remote storage schemes,
+and from having to install CLI tools for each service.
```

Comment:
few typos below:

1. If `dvc unlock` (of if we edit value manually) ...
2. -f option  mentions `dvc run` while it should be `dvc import-url`

Reply:
Thanks. Updated text in 1a214ea